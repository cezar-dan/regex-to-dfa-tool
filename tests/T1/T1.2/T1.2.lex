2
TWO
0
0,'2',1
1,'2',2
1

01
PATTERN
0
0,'0',1
0,'1',2
2,'0',3
2,'1',4
3,'0',6
3,'1',1
4,'0',5
4,'1',4
5,'0',6
5,'1',8
6,'0',3
6,'1',7
7,'0',9
7,'1',1
8,'0',10
8,'1',11
9,'0',1
9,'1',8
10,'0',12
10,'1',13
11,'0',12
11,'1',13
12,'0',12
12,'1',13
13,'0',12
13,'1',13
11 13 12 10