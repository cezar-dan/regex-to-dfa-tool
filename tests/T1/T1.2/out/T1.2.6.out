TWO 2
PATTERN 1000010100
TWO 2
PATTERN 1001011